.buildkite/lint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+bundle install
+
+echo "--- :rubocop: Running Rubocop"
```

Comment:
Off topic, but how cool is it that Buildkite has support for all sorts of tooling emojis? 🤩